### PR TITLE
Add hcloud-cli Tutorial

### PR DESCRIPTION
**Pull request by LKaemmerling**
We plan to add a hcloud-cli tutorial for the start.

### CODE REVIEW

File: tutorials/hcloud-howto-cli.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
+## Introduction
+
+**Prerequisites**
+
+* Hetzner Cloud API Token 
+
+  * Basic knowledge about the Hetzner Cloud
+       * We assume that you know, what a server, an image, a server type or a volume is.
+  * Visit Hetzner Cloud Console at [https://console.hetzner.cloud](https://console.hetzner.cloud), select your project, and create a new API Token.
+* hcloud-ci is installed
```
Comment (n-se) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
* hcloud-cli is installed
```

File: tutorials/hcloud-howto-cli.md [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+  * Windows, FreeBSD
+        * Grab your pre-built binary from [Github](https://github.com/hetznercloud/cli/releases/latest)
+  * Linux
+        * Using [Linuxbrew](http://linuxbrew.sh/)
```
Comment (n-se) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
        * Using [Linuxbrew](https://linuxbrew.sh/)
```

File: tutorials/hcloud-howto-cli.md [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+              * `brew install hcloud`
+        * Grab your pre-built binary from [Github](https://github.com/hetznercloud/cli/releases/latest)
+  * MacOS
+        * Using [Homebrew](http://homebrew.sh/)
```
Comment (n-se) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
        * Using [Homebrew](https://brew.sh/)
```

Comment (n-se) [reply tutorials/hcloud-howto-cli.md]: https://homebrew.sh/ is parked somewhere and forwards to unserious website

File: tutorials/hcloud-howto-cli.md [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+
+The output should be similar to:
+
+```
+ACTIVE   NAME
+*        my-super-project
+```
+
+
+Congratulations! You have created your first context, let' go to the next step!
```
Comment (n-se) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
Congratulations! You have created your first context, let's go to the next step!
```

File: tutorials/hcloud-howto-cli.md [hunk 5]
```diff
@@ -0,0 +1,351 @@
...
+```
+We want to create the cheapest server, a `cx11`. Now just save this information in your mind, you will need it later!
+
+Then we need an image, which should be the base of the server. The Hetzner Cloud supports a various range of images. You can list all available images with: `hcloud image list`
+
+You can grab any image you want from there, we will choose `ubuntu-18.04` for this tutorial.
+
+So now you have all values which are needed for creating a basic server!
+
+The command to create a server is `hcloud server create`, so if you want to create a server with the server type `cx11` and the image `ubuntu-18.04` just run the command in your console
```
Comment (n-se) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
The command to create a server is `hcloud server create`, so if you want to create a server with the server type `cx11` and the image `ubuntu-18.04` just run the following command in your console
```

File: tutorials/hcloud-howto-cli.md [hunk 6]
```diff
@@ -0,0 +1,351 @@
...
+With `hcloud server describe` you can see all available information about your server.
+
+```bash
+hcloud server describe my-cool-server
+```
+
+You can now see all the information about the server:
+
+```
+ID:        2020522
```
Comment (LKaemmerling) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
ID:        1234
```

File: tutorials/hcloud-howto-cli.md [hunk 7]
```diff
@@ -0,0 +1,351 @@
...
+
+Cool! You now created a server called `my-cool-server`! After the creation finished you should see a similar output like:
+
+```
+Waiting for server 1234 to have started... done
+Server 1234 created
+IPv4: <10.0.0.1>
+Root password: TmRsdC3NCgHAbnf9f3rq
+```
+You can now connect to your server via SSH with the Root password!
```
Comment (frommMoritz) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
You can now connect to your server via SSH with the Root password! Please note, for security purposes it's always recommended to [setup key based ssh-access](https://www.cyberciti.biz/faq/how-to-set-up-ssh-keys-on-linux-unix/).
```

File: tutorials/hcloud-howto-cli.md [hunk 8]
```diff
@@ -0,0 +1,351 @@
...
+  ID:        1
+  Name:        cx11
+  Description:    CX11
+  Cores:    1
+  Memory:    2 GB
+  Disk:        20 GB
+  Storage Type:    local
+Public Net:
+  IPv4:
+    IP:        116.203.101.10
```
Comment (frommMoritz) [inline tutorials/hcloud-howto-cli.md]: In the other parts of this tutorial the IP is usually replaced by `<10.0.0.1>`(IPv4) or `<2001:db8::/64>` (IPv6), but in this section it's the normal IP. 

```suggestion
    IP:        <10.0.0.1>
```

File: tutorials/hcloud-howto-cli.md [hunk 9]
```diff
@@ -0,0 +1,351 @@
...
+    Blocked:    no
+    DNS:    static.10.101.203.116.clients.your-server.de
+  IPv6:
+    IP:        2a01:4f8:1c1c:e101::/64
```
Comment (frommMoritz) [inline tutorials/hcloud-howto-cli.md]:
```suggestion
    IP:        <2001:db8::/64>
```

**Pull request merged**